cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/logging/Log.java

```diff
@@ -125,14 +188,23 @@ private static void log(Level level, Object arg0, Object... args) {
      */
     public static void setLogLevel(Level level) {
         // configure own logger
-        logger.setLevel(level);
-        Arrays.stream(logger.getHandlers()).forEach(h -> h.setLevel(level));
+        baseLogger.setLevel(level);
+
+        // configure child loggers' loglevel
+        // otherwise, they won't emit the log records correctly
+        for (String logName : logInstances.keySet()) {
+            Logger logger = logInstances.get(logName).logger;
+            logger.setLevel(level);
+        }
```

Comment:
Could be expressed easier more performant by iterating over the values of the map:
```suggestion
        for (Log log : logInstances.values()) {
            log.logger.setLevel(level);
        }
```

Reply:
That code was more for debugging initially. I'll clean it up